backstop/scenarios/button.js

```diff
@@ -15,9 +15,11 @@ module.exports = [
   }),
   scenario('Grouped Default', {
     selectors: ['#demo-grouped-default'],
+    requireSameDimensions: false,
```

Comment:
Why not just replace test images?

Reply:
hmm, good point. I don't like replacing test images in general